cpp/src/svm/kernelcache.cuh

```diff
@@ -132,12 +132,13 @@ class KernelCache {
     MLCommon::LinAlg::range(k_col_idx.data(), n_ws, stream);
 
     // Init cub buffers
-    size_t bytes1, bytes2;
+    std::size_t bytes1{};
+    std::size_t bytes2{};
     cub::DeviceRadixSort::SortKeys(
       NULL, bytes1, unique_idx.data(), unique_idx.data(), n_ws, 0, sizeof(int) * 8, stream);
     cub::DeviceSelect::Unique(
       NULL, bytes2, unique_idx.data(), unique_idx.data(), d_num_selected_out.data(), n_ws, stream);
-    d_temp_storage_size = max(bytes1, bytes2);
+    d_temp_storage_size = std::max(bytes1, bytes2);
```

Comment:
Don't forget to `#include <algorithm>` to pick this up

Reply:
Done.